docs/getting-started/reference/glossary.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Glossary
+---
+
+# Waku Docs Glossary
+
+Definitions and usage of the terminology used in the Waku ecosystem.
+
+### Bootstrapping
+
+Bootstrapping is the initial [peer discovery](#peer-discovery) process where new [nodes](#node) on the network locate and establish connections with one another.
+
+### Content Topic
+
+A content topic is a string attached to [messages](#waku-message) to enable [protocol-level](#protocol) features like selective message processing and retrieval based on specific criteria.
+
+### Dappnode
+
+Dappnode is an open-source platform that simplifies the hosting and management of decentralized applications and blockchain nodes, including [Waku](#waku).
+
+### Discv5
+
+Discv5 is a [peer discovery](#peer-discovery) mechanism using a Distributed Hash Table (DHT) to store [ENR](#enr) records, providing censorship resistance, load distribution, and enhanced network resilience.
+
+### DNS Discovery
+
+DNS discovery is a [peer discovery](#peer-discovery) mechanism that allows the registration of an [ENR](#enr) tree in the TXT field of a domain name, enabling the storage of [node](#node) connection details and promoting decentralization.
+
+### ENR
+
+Ethereum Node Record (ENR) is a specification used to represent and identify [nodes](#node), facilitating [discovery](#peer-discovery) and communication within the network.
+
+### Filter
+
+Filter is a [protocol](#protocol) that enables [light nodes](#light-node) to selectively subscribe to specific [messages](#waku-message) transmitted by [peers](#peer) using [content topics](#content-topic).
```

Comment:
I would mention that this is an cheap access or light access to the Relay Network.